core/src/servico/ProfessorService.java

```diff
@@ -65,38 +65,29 @@ public ProfessorService(){
         
         List<ProfessorModel> profsPossiveis = new ArrayList<ProfessorModel>();
         
-        for(Professor professor : todosProfessores){
-            
-            if(!professor.equals(professorAusente)){
-                //profsMenosAusente.add(professor);
-                
-                if(professor.EhCompativelCom(aulasPerdidas)){
-                    
-                    Boolean compatível = true;
-                    
-                    for(Ausencia ausenciaPreExistente : todasAsAusencias)
-                    {
-                        if(ausenciaPreExistente.getProfessorSubstituto() != null)
-                        {
-                            if(ausenciaPreExistente.getProfessorSubstituto().getNome().equals(professor.getNome()))
-                            {
-                                if(ausenciaPreExistente.getPeriodo().overlaps(ausência.getPeriodo()))
-                                    compatível = false;
-                            }
-                        }
-                    }
-                    
-                    if(compatível)
-                    {
-                        ProfessorModel model = new ProfessorModel();
+        for(Professor professor : todosProfessores) {
+            if(professor.equals(professorAusente) || !professor.EhCompativelCom(aulasPerdidas))

```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Line is longer than 80 characters (found 95).](https://www.codacy.com/app/calebepb/substituicao/file/2734536146/issues/source?bid=3308597&fileBranchId=3351373#l69)